examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -8023,10 +8023,10 @@
               "mfgCode": null,
               "side": "server",
               "included": 1,
-              "storageOption": "RAM",
+              "storageOption": "NVM",
               "singleton": 0,
               "bounded": 0,
-              "defaultValue": "0xFF",
+              "defaultValue": null,
```

Comment:
Unfortunately, ZAP does not actually support having `null` here so far.  It leads to the default ending up as 0, not null.. Please leave the 0xFF for now until they fix that.

Reply:
StartUpOnOff is an enum whose value can only be 0, 1, or 2. I can no longer set this to 0xFF. I'll change this to 0x1.

Reply:
> I can no longer set this to 0xFF

Can no longer in that ZAP won't let you?

@tecimovic @wehale Could we please get this nullability stuff sorted out?  We could get by for a bit with the workaround of setting the right bit patter, but if ZAP won't let us do that _and_ mishandles its own nullability UI we have a serious problem.

@du48s03 in the meantime, how do you feel about hand-editing the .zap file to put the right value there?  :(

Reply:
Thank you!  The ZAP end is fixed now, and once https://github.com/project-chip/connectedhomeip/pull/16894 merges we won't have to do this manual thing....